src/cargo/ops/cargo_package.rs

```diff
@@ -152,14 +152,14 @@ fn check_metadata(pkg: &Package, config: &Config) -> CargoResult<()> {
 
         config.shell().warn(&format!(
             "manifest has no {things}.\n\
-             See http://doc.crates.io/manifest.html#package-metadata for more info.",
+             See <http://doc.crates.io/manifest.html#package-metadata> for more info.",
```

Comment:
The previous format seems better for humans.

Reply:
I'd like to conform to the style used in Rustc error messages and diagnostics, if you don't mind too much?

Reply:
Sure.  SGTM